src/views/dashboard/playground/docs/getting-started.md

```diff
@@ -172,51 +114,151 @@ Multiple fields can be grouped together. The following SQL statement returns the
 SELECT hostname, environment,
     mean(usage_user) as usage_user_avg
     FROM cpu_metrics
-    GROUP BY hostname, environment;
+    GROUP BY hostname, environment
+    LIMIT 100;
 ```
 
-95 percent usage reflects the peak usage of the CPU. The following SQL statement returns the 95 percent CPU usage group by `hostname` and `environment`.
+95 percent usage reflects the peak usage of the CPU.
+The following SQL statement returns the 95 percent CPU usage group by `hostname` and `environment`.
 
 ```sql
 SELECT hostname, environment,
     approx_percentile_cont(usage_user, 0.95) as usage_user_95,
     approx_percentile_cont(usage_system, 0.95) as usage_system_95,
     approx_percentile_cont(usage_idle, 0.95) as usage_idle_95
     FROM cpu_metrics
-    GROUP BY hostname, environment;
+    GROUP BY hostname, environment
+    LIMIT 100;
 ```
 
 See more about [`GROUP BY` clause](https://docs.greptime.com/reference/sql/group_by).
 
-### Group by Time Intervals
+### Aggregate data within a range of time
```

Comment:
I think we should move p95 example to this part as well. It only makes sense when using p95 within a time window

Reply:
There is currently no data suitable for the p95 example, so I removed the p95 example and consider adding it in the future.